src/modules/schedule/components/calendar/calendar.tsx

```diff
@@ -1,38 +1,38 @@
 import { type FC } from 'react';
 import FullCalendar from '@fullcalendar/react';
 import timeGridPlugin from '@fullcalendar/timegrid';
-import { format } from 'date-fns';
+import { format, getHours, set, getMinutes } from 'date-fns';
 import type { CalendarProps } from './calendar.interface';
+import { trpc } from '@/modules/core/utils/trpc.utils';
 
 import './calendar.scss';
-
-// TODO
-// const today = new Date();
-// const events = [
-//   {
-//     id: '2',
-//     title: 'Событие',
-//     start: new Date(2024, 2, 2, 13, 0, 0),
-//     end: new Date(2024, 2, 2, 14, 0, 0),
-//   },
-//   {
-//     id: '3',
-//     title: 'Событие',
-//     start: new Date(2024, 2, 4, 13, 0, 0),
-//     end: new Date(2024, 2, 4, 14, 0, 0),
-//   },
-//   {
-//     id: '4',
-//     title: 'Событие',
-//     start: new Date(2024, 2, 5, 15, 0, 0),
-//     end: new Date(2024, 2, 5, 16, 0, 0),
-//   },
-// ];
+import styles from './calendarEvent.module.scss';
+import { Typography } from '@/modules/core/components/typogrpahy';
 
 export const Calendar: FC<CalendarProps> = () => {
+  const [me] = trpc.user.me.useSuspenseQuery({ expand: ['professional'] });
+
+  const [events] = trpc.booking.list.useSuspenseQuery({
+    expand: ['serviceProfessional'],
+    professionalId: me.professional?.id,
+  });
+
+  const handleViewMount = ({ el }: { el: HTMLElement }) => {
+    const timeZone = format(new Date(), 'zzzz').replace('GMT', 'GMT ');
+    const timeGridAxis = el.querySelector('.fc-timegrid-axis-frame');
+
+    if (timeGridAxis) {
+      timeGridAxis.innerHTML = `<span style="font-size: 11px">${timeZone}</span>`;
+    }
+  };
```

Comment:
Can you make double check if there is some other way to add timezone? I don't really like the idea using innerHTML for this.

Reply:
This is the only way to insert dynamic content in the right place